scss/components/_state-screen.scss

```diff
@@ -30,7 +30,7 @@
     @extend .oo-expand;
     text-decoration:none;
     cursor: default;
-    z-index: 1000;
+    z-index: $zindex_1000;
```

Comment:
I'm not sure if this has been fixed in a later version, but in my local environment this seems to block the click button of the Discovery Up Next panel. Please make sure to check whether this is the case.

![screen shot 2017-10-12 at 12 28 32 pm](https://user-images.githubusercontent.com/1322503/31515218-35af9dc8-af49-11e7-8956-f18064014d0a.png)

In order to see this you will need to add the Discovery plugin:
//player.ooyala.com/static/v4/candidate/latest/other-plugin/discovery_api.js

And use a video that has Discovery enabled, such as:

ec: l1MzhtNTE6eOf81reieCvXoynnA7TUdp
pbid: e4c115a4f4e74866817e1c7fa60dfc39
pcode: 9odGgyOhmvtSyMbOojUa85Ug4if_

Reply:
oh, thank you! We will create a new task with this issue.